locales/hi/php.json

```diff
@@ -61,7 +61,7 @@
     "lte.file": "The :attribute must be less than or equal :value kilobytes.",
     "lte.numeric": "The :attribute must be less than or equal :value.",
     "lte.string": "The :attribute must be less than or equal :value characters.",
-    "mac_address": "The :attribute must be a valid MAC address.",
+    "mac_address": ":Attribute एक मान्य MAC address होना चाहिए.",
```

Comment:
Is it correct that the sentence ends with a full stop and not the `|` symbol?

Reply:
You are right. In Hindi language, sentence should be ends with `।` symbol. sorry for this silly typo-mistake . Lemme fix it.

Reply:
Make changes, commit and push. So these changes will fall into this PR.